includes/compat/trait-wc-stripe-subscriptions.php

```diff
@@ -987,6 +987,10 @@ public function maybe_render_subscription_payment_method( $payment_method_to_dis
 							/* translators: 1) email address associated with the Stripe Link payment method */
 							$payment_method_to_display = sprintf( __( 'Via Stripe Link (%1$s)', 'woocommerce-gateway-stripe' ), $source->link->email );
 							break 3;
+						case WC_Stripe_Payment_Methods::BACS_DEBIT:
+							/* translators: 1) email address associated with the Stripe Link payment method */
```

Comment:
We may want to update this comment to the following:

```php
/* translators: 1) the Bacs Direct Debit payment method's last 4 numbers */
```

Reply:
done [17ca123](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3987/commits/17ca123d1df579ca7b32b4a85a88477b4e8306da)